guides/developer/getting-set-up.md

```diff
@@ -79,7 +79,7 @@ necessary.
 
 ## IntelliJ
 
-Run `./gradlew idea` in a service's directory to generate IntelliJ project files
+Import the project into IntelliJ by selecting `New` > `Project from Existing Sources` > Navigating to a service's `build.gradle` file. Example: `~/dev/spinnaker/clouddriver/build.gradle`.
```

Comment:
Import  the project into IntelliJ:
1. Select __Project from Existing Sources__ from the __New__ menu.
2. Browse to, and select, the service's `build.gradle` file,
For example:
...